### ISSUE TITLE
Node stuck at block 22159

### ISSUE THREAD

**Issue by flavioroth** [bitclout/run#44]
Hello,

Since a few hours back, my node is stuck on block 22159.

I don't know if that is related but the node keeps complaining about the same duplicate range of blocks going from 22126 to 22126 
```
W0508 14:40:58.916311       1 fZhbf2oz.go:1] Got duplicate block < 21877, 00000000004df1710e3a55bfb006005e07c6b75a3d76509875a270ccbeacd81e > from peer [ Remote Address: 65.21.92.186:17000 PeerID=238 ]
W0508 14:40:58.941549       1 fZhbf2oz.go:1] Got duplicate block < 21878, 000000000027799d66ca7a47fb6f9d88bbe3374fb1ea16ffe395277607ba6977 > from peer [ Remote Address: 65.21.92.186:17000 PeerID=238 ]
W0508 14:40:58.941792       1 fZhbf2oz.go:1] Got duplicate block < 21879, 000000000072ba78d0e34e5ff549b50572420b5ae33556b5079044c6b838be77 > from peer [ Remote Address: 65.21.92.186:17000 PeerID=238 ]

.... a few hundreds similar lines

W0508 14:41:33.190892       1 fZhbf2oz.go:1] Got duplicate block < 22125, 00000000003816ccd939633bae7274b283417323f034bdf137d920bd42d873bd > from peer [ Remote Address: 65.21.92.186:17000 PeerID=238 ]
W0508 14:41:33.191315       1 fZhbf2oz.go:1] Got duplicate block < 22126, 000000000072419a36788fc192d6b959b2ad8289dd6c018f28c27e37fe08ae2e > from peer [ Remote Address: 65.21.92.186:17000 PeerID=238 ]
W0508 14:41:33.225144       1 fZhbf2oz.go:1] Got duplicate block < 21877, 00000000004df1710e3a55bfb006005e07c6b75a3d76509875a270ccbeacd81e > from peer [ Remote Address: 65.21.92.186:17000 PeerID=238 ]
W0508 14:41:33.248887       1 fZhbf2oz.go:1] Got duplicate block < 21878, 000000000027799d66ca7a47fb6f9d88bbe3374fb1ea16ffe395277607ba6977 > from peer [ Remote Address: 65.21.92.186:17000 PeerID=238 ]

```

Edit: Issue still there after pulling the [latest core](https://github.com/orgs/bitclout/packages/container/core-obf/2413971?tag=60fea2dbcb3ab565c2a49ca826eda2bf1161c880).

**Comment by marshj1977** [comment]
Also see https://github.com/bitclout/run/issues/43 ;)

**Comment by ropolexi** [comment]
I also have this problem when running the node. There were lot of duplicate errors and kept on repeating and stuck in a loop. Tried to clean up the docker and tried again from fresh but now the admin panel not showing anymore, this time I disabled txindex since it took huge amount of hard disk space more than 200GB for this few days. But now it is at about 25GB in size. Not sure this txindex and admin panel seems relevant.
